### PR TITLE
updated to fit hashistack 0.7.1

### LINKED ISSUE #71: Bump up to v0.7.x of hashistack-box
## What is the issue?
Upgrade and test v0.7.0 of the hashistack box. 
https://app.vagrantup.com/fredrikhgrelland/boxes/hashistack/versions/0.7.0

## Suggestion(s)/solution(s) [Optional]
Change `Vagrantfile.default` versioning to fetch newest box between 0.7 and 0.8

## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### PR DESCRIPTION
**Pull request by dangernil**
## Closes/fixes/resolves issue(s)?
Closes #71 

## What was added/changed/fixed?
- Updated to fit hashistack v.0.7.0

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### CODE REVIEW

**Comment by zhenik** [comment]
You might face this issue https://github.com/fredrikhgrelland/vagrant-hashistack/issues/441

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +12,7 @@
 
 - Proper rendering of credentials #66
 - Removed box example [no issue]
+- Updated to fit hashistack v.0.7.0 #71
```
Comment (pdmthorsrud) [inline CHANGELOG.md]:
```suggestion
- Updated to fit hashistack v0.7.1 #71
```
or potentially v0.7.x

**Comment by zhenik** [comment]
In `/dev/vagrant/bootstrap/nomad/post/022-update-nomad-write-default-policy.yml`

You need update `path` 
```
cmd: vault kv get -field=secret-id secret/nomad-bootstrap-token
```
to 

```
cmd: vault kv get -field=secret-id secret/example/nomad-bootstrap-token
```

You can check the proper path in the Vault UI
Screenshot 2020-11-06 at 12 36 08

**Review by claesgill — DISMISSED** [review]

**Review by pdmthorsrud — DISMISSED** [review]

**Comment by zhenik** [comment]
Also, @dangernil check here /dev/ansible/05_test_persistence.yml line 11

**Review by claesgill — APPROVED** [review]

**Review by pdmthorsrud — APPROVED** [review]

**Pull request merged**